### DISCUSSION TITLE
Alerts to PSA from Alerts API

### DISCUSSION THREAD

**Discussion by NLygitadm** [KelvinTegelaar/CIPP#1825] [Q&A]
Hello !

I have a recurring message on CIPP for one of the tenants we manage.
I disabled all alerts for this tenant to test but it persists.
I don't understand what this message refers to, especially since the PSA link is not activated for us.
Do you have an idea or a path to explore?

THANKS !

![cippalert](https://github.com/KelvinTegelaar/CIPP/assets/127679542/c53ba850-94bd-4f52-b471-ec2b9897261d)

**Comment by KelvinTegelaar** [comment]
Only sponsors get support
